packages/api/lambdas/process-s3-dead-letter-archive.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const log = require('@cumulus/common/log');
+
+const { s3 } = require('@cumulus/aws-client/services');
+const { getJsonS3Object } = require('@cumulus/aws-client/S3');
+const { getKnexClient } = require('@cumulus/db');
+
+const { writeRecords } = require('./sf-event-sqs-to-db-records');
+
+async function processDeadLetterArchive({
```

Comment:
Particularly given this is .js : Docstring please!

Reply:
7afb74661016d2cdd28a82ac61d8b174107a41ad